app/policies/note_policy.rb

```diff
@@ -0,0 +1,35 @@
+class NotePolicy < ApplicationPolicy
```

Comment:
Should you add `authorize` method to controllers to start using policies?

For example:
```ruby
def authorize_resource!
  authorize! note
end
```